client/my-sites/plans-v2/products-grid-alt/index.tsx

```diff
@@ -8,19 +8,13 @@ import { useSelector } from 'react-redux';
  * Internal dependencies
  */
 import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
-import { getMonthlyPlanByYearly, getYearlyPlanByMonthly } from 'calypso/lib/plans';
-import { JETPACK_LEGACY_PLANS } from 'calypso/lib/plans/constants';
+import * as PlanConstants from 'calypso/lib/plans/constants';
+import * as ProductConstants from 'calypso/lib/products-values/constants';
 import { getCurrentUserCurrencyCode } from 'calypso/state/current-user/selectors';
 import getSitePlan from 'calypso/state/sites/selectors/get-site-plan';
 import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
 import JetpackFreeCard from 'calypso/components/jetpack/card/jetpack-free-card-alt';
-import { SELECTOR_PLANS_ALT } from '../constants';
-import {
-	getAllOptionsFromSlug,
-	getJetpackDescriptionWithOptions,
-	slugToSelectorProduct,
-} from '../utils';
-import useGetPlansGridProducts from '../use-get-plans-grid-products';
```

Comment:
I think we need this hook. Hardcoding the list of cards we want to show won't work if you own a product that doesn't appear in the hardcoded list. For instance, if you own Jetpack Security Real-time or Jetpack Backup Real-time, you won't see that card in the Plans grid.